### PR TITLE
[Duplicate #6214] How to Integrate Django-Tailwind Styling in an Application

### LINKED ISSUE #6018: [Language] How To Integrate Django-Tailwind Styling In Application
### Introduction
Every application requires good stylings for better and readable contents.  Over the years, the Cascadian Styles Sheet,  CSS has dominated before frameworks came into existence. In this tutorial, we will be covering the way to integrate django-tailwind framework into an application.

### Key takeaways
At the end of this tutorial, readers should be able to apply this styling framework into their application. 

### Article Quality
This tutorial will lead readers to the proper way of integrating django-tailwind for the purpose of styling application.

### PR DESCRIPTION
**Pull request by Charity10**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x]Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6018

### CODE REVIEW

**Comment by Charity10** [comment]
I mistakenly deleted the fork on my repository with the thought that I could resolve the issue. So, later I was not able to reference the branch again and had to create another PR. All of the edits are intact, and I have made the corrections sir. Thank you 
@Dawe-7

**Comment by Charity10** [comment]
How would you help me so that this can regain its position back?

**Comment by Charity10** [comment]
Added sir. @Dawe-7

**Review by Dawe-7 — APPROVED** [review]
Hello @Charity10 , The article is an interesting read. Nice job!
Just a few things I'd like to point out that you should be keen on in your future articles:
1. It is always advisable to submit an article that is fully polished - By this, I mean having an article that does not have too many grammatical corrections. I suggest that you check out our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) which has links to various resourceful tools for writing.
2. Ensure all your images are of the size 600x400 pixels and are not blurred - Make use of tools such as [pixlr Image Editor](https://pixlr.com/e/) to resize them.
3. Check on how you save your files - There should be no capitalized letters when creating a branch name in your vscode.
4. Lastly please go through ALL the corrections I have made so that you can not repeat the same mistakes - Ensure that all changes requested are in place. KIndly avoid not effecting all the changes for us to avoid going back and forth in review.

That's it, a final review will be done now by a Content Moderator. Otherwise, congratulations on contributing your first article to the EngEd program.

In case of any question or clarification (currently or in the future), do not hesitate to reach out via Slack (Marsa) or my personal email ([dantedawek@gmail.com](mailto:dantedawek@gmail.com)).
I am ALWAYS here to help.

Looking forward to seeing you contribute more to the EngEd program.

**Comment by Charity10** [comment]
Your reviews gave me awesome experiences been a first timer contributor. I have noted the raised points sir, thank you!

**Comment by Charity10** [comment]
@WanjaMIKE sir, I noticed you are a moderator sir. Can you please merge this PR because it was reopened 8 days ago and ready for final review since then. The PR lost its normal position, which was at bottom of page 3 before.

cc: @hectorkambow

**Comment by hectorkambow** [comment]
@Charity10  

Please reach out to me via a Slack DM 👍

**Comment by LinusMuema** [comment]
jumping on this one

File: content/articles/how-to-integrate-django-tailwind-styling-in-an-application/index.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+Open your git bash, and type in the following commands:
+
+```bash
+cd
+cd Desktop
+mkdir project
+cd project
+```
+Check the working directory and ensure you are on the `Desktop` directory. You also made a directory called `project`, change to it.
```
Comment (LinusMuema) [inline content/articles/how-to-integrate-django-tailwind-styling-in-an-application/index.md:48]: Hi @Dawe-7 

I am noticing cases like these where there is a mix of active and passive voices, which make the sentences illegible. Please change them to active voice as this is more engaging to the users and easier for them to follow through.

A good example:
 _-_ `You also made a directory called project, change to it`
 _+_ `Create a directory called project and navigate to it`

**Review by LinusMuema — CHANGES_REQUESTED** [review]
Hi @Dawe-7 

Please check the review and update the pr once done.

**Comment by Charity10** [comment]
@LinusMuema sir

**Comment by LinusMuema** [comment]
Hi @Charity10 

Thanks for joining section and Welcome. You can go through the commits made during the review to improve on future articles.

cc: @Dawe-7 please check my commits for the changes I have made to improve on your future reviews.

**Pull request merged**